.circleci/config.yml

```diff
@@ -70,10 +75,25 @@ jobs:
             docker load -i ./docker-images/spring-cloud-kubernetes-discoveryserver.tar
             docker load -i ./docker-images/spring-cloud-kubernetes-configserver.tar
       - run:
-          name: Run Kind Integration Tests
+          name: testcontainers integration tests
           command: |
+            # we need sort, because the last project calls K3S::stop
+            # we need tail to exclude root project
+            INTEGRATION_PROJECTS=$(mvn -Dexec.executable='echo' -Dexec.args='${project.artifactId}' exec:exec -q | tail -n +2 | sort)
+            cd spring-cloud-kubernetes-test-support
+            mvn clean install
+            cd ..
+
             cd spring-cloud-kubernetes-integration-tests
-            ./run.sh
+            PROJECT_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+            for p in "${INTEGRATION_PROJECTS[@]}"; do
+              cd "${p}"
+              mvn spring-boot:build-image \
+              	  -Dspring-boot.build-image.imageName=docker.io/springcloud/$p:${PROJECT_VERSION} \
+                  -Dspring-boot.build-image.builder=paketobuildpacks/builder
+              mvn clean verify -DexcludeITTests=
```

Comment:
What is this doing?

Reply:
you are correct again, as a matter of fact, we now run those tests with simply `mvn clean install`. I'll comment where it needs your eyes on how this is done